### DISCUSSION TITLE
Hiding orderList controls

### DISCUSSION THREAD

**Discussion by pnahorniukATS** [primefaces/community#1707] [PrimeNG] [answered]
Hello,
In our project based on Angular v16.2.12 we have been using primeng v16.7.1.
It would be great to have an option to hide controls for orderList component, so I have just a list of items that I could drag and drop to reorder them.

I know there is a way to achieve that hiding controls via CSS. But, for me, having unnecessary elements in DOM feels wrong.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, it's not supported right now, however, you can create a PR to make this possible.
